README.md

```diff
@@ -192,27 +192,39 @@ In the event that the file does not have an extension or the extension is unreco
 
 ### Instance methods (on `explorer`)
 
-#### `search([searchPath])`
+#### `search([searchPath][, searchOptions])`
+
 
 Searches for a configuration file. Returns a Promise that resolves with `null`, if nothing is found, or an object with two properties:
 - `config`: The loaded and parsed configuration.
 - `filepath`: The filepath where this configuration was found.
 
+When the [search option](#searchoptions) `ignoreEmpty` is set to false and an empty configuration file is found, the result will have `config` set to undefined and an additional property `isEmpty` set to true.
+
 ```js
 explorer.search() // searches from process.cwd()
 explorer.search('start/search/here');
 explorer.search('start/search/at/this/file.css');
+
+// will stop on an empty configuration file
+explorer.search('start/search/here', { ignoreEmpty: false })
 ```
 
 If you provide `searchPath`, cosmiconfig will start its search at `searchPath` and continue to search up the directory tree, as documented above.
 By default, `searchPath` is set to `process.cwd()`.
 
+##### searchOptions
```

Comment:
I used the header to link from above

> When the [search option](#searchoptions) `ignoreEmpty` is set to false

I did this to ensure we weren't just dropping a term that hasn't been introduced yet, and thought it would be odd to place the definition above the paragraph where it was linked.  I'm open to a reorganization though

Reply:
Okay, that makes sense. Just removing the bulletting and adding a newline after the heading should do.